tests/network/macvtap.go

```diff
@@ -103,13 +102,12 @@ var _ = SIGDescribe("Macvtap", decorators.Macvtap, func() {
 	createAlpineVMIStaticIPOnNode := func(nodeName string, networkName string, ifaceName string, ipCIDR string, mac *string) *v1.VirtualMachineInstance {
 		var vmi *v1.VirtualMachineInstance
 		if mac != nil {
-			vmi = newAlpineVMIWithExplicitMac(networkName, *mac)
+			vmi = newAlpineVMIWithExplicitMac(networkName, *mac, libvmi.WithNodeAffinityFor(nodeName))
 		} else {
-			vmi = newAlpineVMIWithMacvtapNetwork(networkName)
+			vmi = newAlpineVMIWithMacvtapNetwork(networkName, libvmi.WithNodeAffinityFor(nodeName))
 		}
-		vmi = libwait.WaitUntilVMIReady(
-			tests.CreateVmiOnNode(vmi, nodeName),
-			console.LoginToAlpine)
+		vmi = tests.RunVMI(vmi, 90)
```

Comment:
A direct creation of the VMI should be enough. I do not understand why there is a need to retry the creation for 90sec. And it was not there before.

Reply:
I used the widely used `RunVMI` function, but it is ok for me to switch toward single creation function call.

Reply:
Widely used does not assure it is correct.
Someone needs a good reason to retry creation failure IMO.

The fact that it was not needed before here, hints it is not a proper replacement.